lib/src/core/room.dart

```diff
@@ -62,9 +62,9 @@ class Room extends DisposableChangeNotifier with EventsEmittable<RoomEvent> {
   RoomOptions get roomOptions => engine.roomOptions;
 
   /// map of SID to RemoteParticipant
-  UnmodifiableMapView<String, RemoteParticipant> get participants =>
-      UnmodifiableMapView(_participants);
-  final _participants = <String, RemoteParticipant>{};
+  UnmodifiableMapView<String, RemoteParticipant> get remoteParticipants =>
```

Comment:
As `remoteParticipants` is publicly accessible, I think we'd want to switch to mapping them by `identites` here instead of `sid`. 
cc @davidzhao for confirmation

Reply:
ah, let me check.

Reply:
ah yes, great catch!

Reply:
Resolved https://github.com/livekit/client-sdk-flutter/pull/435/commits/bfe138f0e4d1acb9c91e62a1edb423b6d3653410